### PR TITLE
[zh-cn] sync blog: 2023-08-21-job-update-post.md

### PR DESCRIPTION
**Pull request by my-git9**

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-08-21-job-update-post.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+We would not have been able to achieve either of these features without Aldo
+Culquicondor (Google) providing excellent domain knowledge and expertise
+throughout the Kubernetes ecosystem.
+-->
+## 致谢
+
+与其他 Kubernetes 特性一样，该特性从测试和修复 Bug 到代码审查，很多人为此项工作做出了贡献。
+
+如果没有 Aldo Culquicondor（Google）在整个 Kubernetes 生态系统中提供出色的领域知识和专业知识，
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2023-08-21-job-update-post.md:372]:
```suggestion
如果没有 Aldo Culquicondor（Google）提供出色的领域知识和跨整个 Kubernetes 生态系统的知识，
```

File: content/zh-cn/blog/_posts/2023-08-21-job-update-post.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+## How can you learn more?
+
+- Read the user-facing documentation for [Pod replacement policy](/docs/concepts/workloads/controllers/job/#pod-replacement-policy),
+[Backoff limit per index](/docs/concepts/workloads/controllers/job/#backoff-limit-per-index), and
+[Pod failure policy](/docs/concepts/workloads/controllers/job/#pod-failure-policy)
+- Read the KEPs for [Pod Replacement Policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3939-allow-replacement-when-fully-terminated),
+[Backoff limit per index](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs), and
+[Pod failure policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3329-retriable-and-non-retriable-failures).
+-->
+## 如何进一步了解
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2023-08-21-job-update-post.md:325]:
```suggestion
## 如何进一步了解 {#how-can-you-learn-more}
```

File: content/zh-cn/blog/_posts/2023-08-21-job-update-post.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+[batch working group](https://github.com/kubernetes/community/tree/master/wg-batch).
+Working groups are relatively short-lived initiatives focused on specific goals.
+The goal of the WG Batch is to improve experience for batch workload users, offer support for
+batch processing use cases, and enhance the
+Job API for common use cases.  If that interests you, please join the working
+group either by subscriping to our
+[mailing list](https://groups.google.com/a/kubernetes.io/g/wg-batch) or on
+[Slack](https://kubernetes.slack.com/messages/wg-batch).
+-->
+## 参与其中
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2023-08-21-job-update-post.md:348]:
```suggestion
## 参与其中 {#getting-Involved}
```

File: content/zh-cn/blog/_posts/2023-08-21-job-update-post.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+## Acknowledgments
+
+As with any Kubernetes feature, multiple people contributed to getting this
+done, from testing and filing bugs to reviewing code.
+
+We would not have been able to achieve either of these features without Aldo
+Culquicondor (Google) providing excellent domain knowledge and expertise
+throughout the Kubernetes ecosystem.
+-->
+## 致谢
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2023-08-21-job-update-post.md:368]:
```suggestion
## 致谢 {#acknowledgments}
```

File: content/zh-cn/blog/_posts/2023-08-21-job-update-post.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+-->
+## 参与其中
+
+这些功能由 [SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps) 赞助。
+社区正在为[批处理工作组](https://github.com/kubernetes/community/tree/master/wg-batch)中的
+Kubernetes 用户积极改进批处理场景。
+工作组是相对短暂的举措，专注于特定目标。WG Batch 的目标是改善批处理工作负载的用户体验、
+提供对批处理场景的支持并增强常见场景下的 Job API。
+如果你对此感兴趣，请通过订阅我们的[邮件列表](https://groups.google.com/a/kubernetes.io/g/wg-batch)或通过
+[Slack](https: http://kubernetes.slack.com/messages/wg-batch) 加入进来。
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2023-08-21-job-update-post.md:356]:
```suggestion
[Slack](https://kubernetes.slack.com/messages/wg-batch)) 加入进来。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**